src/components/SizeObserver/SizeObserver.test.tsx

```diff
@@ -0,0 +1,104 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import * as React from 'react';
+import { mount, ReactWrapper } from 'enzyme';
+import SizeObserver, { SizeObserverProps } from './index';
+
+describe('<SizeObserver />', () => {
+  let component: ReactWrapper<SizeObserverProps>;
+  let onSizeChange: ResizeObserverCallback;
+
+  beforeEach(() => {
+    // tslint:disable-next-line:no-function-expression
+    (window as any).ResizeObserver = function(callback: ResizeObserverCallback) {
+      onSizeChange = callback;
+    };
+    ResizeObserver.prototype.disconnect = jest.fn();
+    ResizeObserver.prototype.observe = jest.fn();
+    ResizeObserver.prototype.unobserve = jest.fn();
+
+    const query = {
+      small: {
+        maxWidth: 500,
+        maxHeight: 500,
+      },
+      large: {
+        minWidth: 501,
+        minHeight: 501,
+      },
+    };
+
+    const renderAt = jest.fn((size?: string) => {
+      switch (size) {
+        case 'large':
+          return <span>Large</span>;
+        case 'small':
+          return <span>Small</span>;
+        default:
+          return <span>Other</span>;
+      }
+    });
+
+    component = mount(<SizeObserver query={query} renderAt={renderAt} />);
```

Comment:
Looks to me like you'd get the same coverage, with clearer snapshots by just looking at the `span`.

```    
component = mount(<SizeObserver query={query} renderAt={renderAt} />).find('span');
```

Reply:
Or maybe `find('div')` to get the wapping div too?

Reply:
Agreed. I will update.